src/Evolution/Systems/ScalarWave/EnergyDensity.hpp

```diff
@@ -37,5 +38,21 @@ template <size_t SpatialDim>
 Scalar<DataVector> energy_density(
     const Scalar<DataVector>& pi,
     const tnsr::i<DataVector, SpatialDim, Frame::Inertial>& phi) noexcept;
+
```

Comment:
following up on @wthrowe's comment about the doxygen grouping. Here you want:
```cpp
     const tnsr::i<DataVector, SpatialDim, Frame::Inertial>& phi) noexcept;
/// @}

namespace Tags {
/// \brief Computes the energy density using ScalarWave::energy_density()
template <size_t SpatialDim>
struct EnergyDensityCompute...
```
and then remove the `/// @}` on line 56 below